backend/oasst_backend/models/text_labels.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+from typing import Optional
+from uuid import UUID, uuid4
+
+import sqlalchemy as sa
+import sqlalchemy.dialects.postgresql as pg
+from sqlmodel import Field, SQLModel
+
+
+class TextLabels(SQLModel, table=True):
+    __tablename__ = "text_labels"
+
+    id: Optional[UUID] = Field(
+        sa_column=sa.Column(
+            pg.UUID(as_uuid=True), primary_key=True, default=uuid4, server_default=sa.text("gen_random_uuid()")
+        ),
+    )
+    created_date: Optional[datetime] = Field(
+        sa_column=sa.Column(sa.DateTime(), nullable=False, server_default=sa.func.current_timestamp()),
+    )
+    api_client_id: UUID = Field(nullable=False, foreign_key="api_client.id")
+    text: str = Field(nullable=False, max_length=2**16)
+    post_id: Optional[UUID] = Field(sa_column=sa.Column(pg.UUID(as_uuid=True), sa.ForeignKey("post.id"), nullable=True))
+    labels: dict[str, float] = Field(default={}, sa_column=sa.Column(pg.JSONB), nullable=False)
```

Comment:
yes I've seen some nulls in the database, but what do you suggest? I'd still leave it `nullable=False` to communicate intent to the programmer

Reply:
ok, we leave it as is it. Just wanted to note that SQLModel will allow submitting `None` and from python there is currently no difference between storing `'null'::jsonb` (which happens when you submit None to non-nullable jsonb column) ... but technically it is of course not the db native null value and it will behave accordingly, e.g. equals to `'null'::jsonb` but not `is null` is false ...